cpp/gcc/Makefile

```diff
@@ -9,12 +9,26 @@ LIB_DIR=	$(SOFT_DEV_DIR)/../ext_lib
 
 MODE ?= 0
 
-#CC  = gcc
-#CXX = g++
-CC  = cc
-CXX = c++
 
-SRW_SRC_DEF=	-D_GNU_SOURCE -D__USE_XOPEN2K8 -DFFTW_ENABLE_FLOAT -D_GM_WITHOUT_BASE -DSRWLIB_STATIC -DNO_TIMER -DANSI_DECLARATORS -DTRILIBRARY -DLINUX
+CC=cc
+CXX=c++
```

Comment:
FYI: I had to exclude these 2 lines via a patch for the conda recipe, to make it built with the conda compilers - https://github.com/nsls-ii-forge/srw-feedstock/blob/master/recipe/compilers.patch.

Reply:
I need to switch the definitions of `CC` and `CXX` to be only if not defined before.
If building for something other than Linux you will have issues with the `SRW_SRC_DEF` variable since it defines `-DLINUX`... I will put an update here to make `CC` and `CXX` to be only if not defined.